projects/packages/publicize/webpack.config.js

```diff
@@ -0,0 +1,55 @@
+const path = require( 'path' );
```

Comment:
It's a shame that we have to add all of this config for just for the classic editor script, but I suppose it can't be helped!